src/views/board.handlebars

```diff
@@ -66,7 +66,7 @@
         {{/each}}
         <form id="new-list" method="POST" action="/board/{{board._id}}">
             <label for="newListNameInput" hidden>List: </label>
-            <input type="text" name="listName" id="newListNameInput">
+            <input type="text" name="listName" id="newListNameInput" required/>
```

Comment:
I don' think the slash is necessary after required, but if it works it doesn't matter